v0/destinations/rockerbox/transform.js

```diff
@@ -0,0 +1,120 @@
+const get = require("get-value");
+
+const set = require("set-value");
+const {
+  defaultRequestConfig,
+  CustomError,
+  removeUndefinedAndNullValues,
+  constructPayload,
+  getSuccessRespEvents,
+  getErrorRespEvents,
+  getHashFromArray
+} = require("../../util");
+const { EventType } = require("../../../constants");
+const { CONFIG_CATEGORIES, MAPPING_CONFIG } = require("./config");
+
+const responseBuilderSimple = (message, category, destination) => {
+  const payload = constructPayload(message, MAPPING_CONFIG[category.name]);
+  // conversion_source is explicitly set to RudderStack
+  payload.conversion_source = "RudderStack";
+
+  const { advertiserId, eventsMap } = destination.Config;
+  const eventsHashMap = getHashFromArray(eventsMap);
+
+  if (!eventsHashMap[message.event.toLowerCase()]) {
+    throw new CustomError(
+      "The event is not associated to a RockerBox event. Aborting!",
+      400
+    );
+  } else {
+    set((payload.action = eventsHashMap[message.event.toLowerCase()]));
```

Comment:
```suggestion
    payload.action = eventsHashMap[message.event.toLowerCase()];
```
No need to use set here. Also the use was incorrect

Reply:
fixed